README.md

```diff
@@ -486,6 +516,190 @@ The supported `PlayerView` events for subtitles are:
 
 You might check out a complete subtitle example in the [`example/`](https://github.com/bitmovin/bitmovin-player-react-native/tree/development/example) app.
 
+### Enabling Picture in Picture mode
+
+In order to make use of the Picture in Picture functionalities provided by the player, it's first necessary to configure your native application to properly support PiP.
+
+The steps required for each platform are described below:
+
+#### Android
+
+**Declare Picture in Picture support on AndroidManifest.xml**
+
+Open `android/app/src/main/AndroidManifest.xml` and set `android:supportsPictureInPicture` to `true`
+on your main activity's manifest. Also, specify that your activity handles layout configuration changes
+so that your activity doesn't relaunch when layout changes occur during PiP mode transitions:
+
+```xml
+<activity android:name=".MainActivity"
+    android:supportsPictureInPicture="true"
+    android:configChanges=
+        "screenSize|smallestScreenSize|screenLayout|orientation"
+    ...
+```
+
+#### iOS
```

Comment:
Also this enabled PiP for tvOS as well, right? (The configs are the same on our API)

Reply:
I think it's good to clarify that tvOS is also supported.

But I couldn't test whether PiP for tvOS was enabled because I don't have a physical device to test on 😕

Reply:
UPDATE: committed to `development`